src/algorithm/PointLocation.cpp

```diff
@@ -42,7 +58,7 @@ PointLocation::isOnLine(const geom::CoordinateXY& p, const geom::CoordinateSeque
     const geom::CoordinateXY* pp = &(pt->getAt<geom::CoordinateXY>(0));
```

Comment:
Maybe change this to a reference so you dispense with the `*` further down,

Reply:
I tried that but was getting compile errors.  Does it need to be a pointer because it gets revalued at the end of the loop?

Reply:
Another option is to simply define references for both points each time through the loop.

Reply:
Oh yeah, const references hate being redefined. 🤷  Nm.